### ISSUE TITLE
Incompatible with any shaderpacks

### ISSUE THREAD

**Issue by ChromaPIE** [Gregicality/gregicality#166]
**Gregicality-1.12.2-0.20.6-hotfix.3**

Crashes after entering a world and starting to render. Crash-report: https://hastebin.com/ozecibaduv.rb

Then I tried to make CCL tolerate the issue making crashes by setting `catchBlockRenderExceptions` **true**, and I could successfully join a world, the shader was rendered just fine except a warning saying that CCL caught an exception kept jumping in at the bottom left corner of the screen. Then I tried to place machines from GregTech and Thermal Expansion, which both requires CCL as a dependency. The first machine I placed was rendered successfully, but when I put another machine, both suddenly turned black. At the first few trials, breaking one machine would make another back to normal, but at last, all machines couldn't be rendered correctly anymore, all are black.

**Comment by hjae78** [comment]
Yup, known issue, from log looks like GTCE issue, has to do with surface rocks
